core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -140,6 +140,17 @@ public DoraMeta.FileStatus load(String path) throws IOException {
 
     mPageSize = Configuration.global().getBytes(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE);
     mBlockMasterClientPool = new BlockMasterClientPool();
+
+    String dbDir = Configuration.getString(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR);
+    Duration duration = Configuration.getDuration(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_TTL);
+    long ttl = duration.isZero() || duration.isNegative() ? -1 : duration.getSeconds();
+    try {
+      mMetaStore = new RocksDBDoraMetaStore(dbDir, ttl);
+    } catch (RuntimeException e) {
+      LOG.error("Cannot init RocksDBDoraMetaStore. Continue without MetaStore", e);
+      mMetaStore = null;
```

Comment:
still work without metaStore?

Reply:
Worker metastore in persistent RocksDB is used to cache FileStatus.
It is a cache.
We have in-memory cache too, `mUfsStatusCache`.
If the RocksDB does not work, the in-memory still works independently.

Reply:
If it exists, will it be double cached?

Reply:
Yes. We configure the in-memory cache to have a small number of items and the RocksDB to have relatively large number of items.
They also have different cache expiring TTL.